### ISSUE TITLE
🐛 `useSortedProperties` fails with unknown CSS properties

### ISSUE THREAD

**Issue by CathalMullan** [biomejs/biome#6105]
### Environment information

```block
CLI:
  Version:                      2.0.0-beta.5
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           linux

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  NO_COLOR:                     unset
  TERM:                         xterm-ghostty
  JS_RUNTIME_VERSION:           unset
  JS_RUNTIME_NAME:              unset
  NODE_PACKAGE_MANAGER:         unset

Biome Configuration:
  Status:                       Loaded successfully
  Path:                         biome.jsonc
  Formatter enabled:            true
  Linter enabled:               true
  Assist enabled:               true
  VCS enabled:                  true

Workspace:
  Open Documents:               0
```

### What happened?

Repro repo: https://github.com/CathalMullan/biome-repro-usesortedproperties

The `useSortedProperties` rule currently errors if there's a property it doesn't know how to order.

```css
.foo {
  position-anchor: --position-anchor;
}
```

```
index.css:1:6 assist/source/useSortedProperties ━━━━━━━━━━━━━━━━━━━━━

  ✖ The properties are not sorted.

  > 1 │ .foo {
      │      ^
  > 2 │   position-anchor: --position-anchor;
  > 3 │ }
      │ ^
    4 │
```

### Expected result

`stylelint-config-recess-order` just skips over unknown properties entirely. 

It will accept all of the following orders, since it doesn't have a understanding of `position-anchor`.

```css
.foo {
  position-anchor: --position-anchor;
}

.bar {
  position-anchor: --position-anchor;
  position: relative;
  box-sizing: border-box;
}

.baz {
  position: relative;
  position-anchor: --position-anchor;
  box-sizing: border-box;
}

.qux {
  position: relative;
  box-sizing: border-box;
  position-anchor: --position-anchor;
}
```

Seems like Biome should copy Stylelints approach, and just skip unknown properties.

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct
